src/proto/types.rs

```diff
@@ -116,7 +116,7 @@ impl CloseCode {
 
 impl CloseCode {
     /// Whether the close code is allowed to be sent over the wire.
-    pub(super) fn is_sendable(self) -> bool {
+    pub fn is_sendable(self) -> bool {
```

Comment:
I think if we're making this public, we should rename it to `is_reserved` to match the naming in the RFC and invert the logic.